wear/src/main/java/com/eveningoutpost/dexdrip/Services/CustomComplicationProviderService.java

```diff
@@ -129,34 +132,56 @@ public void onComplicationUpdate(
                 is_stale = true;
             }
         }
+
+        //Loop status by @gregorybel
+        String externalStatusString = PersistentStore.getString("remote-status-string");
+
         Log.d(TAG, "Returning complication text: " + numberText);
+        Log.d(TAG, "Returning complication status: " + externalStatusString);
 
         COMPLICATION_STATE state = COMPLICATION_STATE.get_enum((int) PersistentStore.getLong(ComplicationTapBroadcastReceiver.COMPLICATION_STORE));
         if (state == null) state = COMPLICATION_STATE.DELTA;
 
         ComplicationData complicationData = null;
-
         final boolean doMgdl = Home.getPreferencesStringWithDefault("units", "mgdl").equals("mgdl");
+        String deltaText = (!is_stale ? (bgReading != null ? unitizedDeltaString(false, false, Home.get_follower(), doMgdl) : "null") : "");
```

Comment:
As this is now reused can we make it the response from a method so that it isn't calculated needlessly if the SHORT + AGO option is being used?